components/esp_hw_support/mac_addr.c

```diff
@@ -86,7 +86,8 @@ esp_err_t esp_efuse_mac_get_custom(uint8_t *mac)
     uint8_t version;
     esp_efuse_read_field_blob(ESP_EFUSE_MAC_CUSTOM_VER, &version, 8);
     if (version != 1) {
-        ESP_LOGE(TAG, "Base MAC address from BLK3 of EFUSE version error, version = %d", version);
+        // version 0 means has not been setup
+        ESP_LOG_LEVEL_LOCAL((version == 0 ? ESP_LOG_DEBUG : ESP_LOG_ERROR), TAG, "Base MAC address from BLK3 of EFUSE version error, version = %d", version);
```

Comment:
Instead of a ternary operator I would suggest a (slightly more verbose but readable)
```c++
if (version == 0) {
    ESP_LOGD(TAG, "No base MAC address in eFuse (version=0)");
} else if (version != 1) {
    ESP_LOGE(TAG, "Base MAC address version error, version = %d", version);
}
```

(i have removed "from BLK3" since that is an ESP32-specific leftover; other chips may have custom MAC address written in another eFuse block)